packages/ui-app/src/Params/Param/findComponent.ts

```diff
@@ -14,12 +14,13 @@ import Hash from './Hash';
 import KeyValue from './KeyValue';
 import KeyValueStorageArray from './KeyValueStorageArray';
 import StringParam from './String';
+import StringBalance from './StringBalance';
 import Unknown from './Unknown';
 import VoteThreshold from './VoteThreshold';
 
 const components: ComponentMap = {
   'AccountId': Account,
-  'Balance': Amount,
+  'Balance': StringBalance,
```

Comment:
`BalanceInput`

Reply:
i've renamed it to `Balance` as discussed